code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -172,6 +173,7 @@
 			beaker.loc = src
 			update_icon(UPDATE_ICON_STATE)
 			updateUsrDialog()
```

Comment:
Remove all these `updateUsrDialog`s please

Reply:
Have all been removed and replaced with SStgui.update_uis